### PR TITLE
Adding new candidate JMeter

### PR DESCRIPTION
**Pull request by xshyamx**
Adding new candidate jmeter

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/JMeterMigration.scala
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/JMeterMigration.scala:30]: I think this is not needed

**Review by eddumelendez — DISMISSED** [review]
Thanks @xshyamx ! This PR looks good to me. Also, I checked the structure downloading jmeter and it meets the requirements. But will wait for @marc0der approvals before merge it

File: src/main/scala/io/sdkman/changelogs/JMeterMigration.scala
Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/JMeterMigration.scala]: Updated to remove it

File: src/main/scala/io/sdkman/changelogs/JMeterMigration.scala
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JMeterMigration.scala:53]: No need for the `case` statement here. You can simply:
```
.map { version ->
    Version(
        ....
        version = version
        ...
    )
}
```

**Review by marc0der — APPROVED** [review]
LGTM! One tiny change requested.

File: src/main/scala/io/sdkman/changelogs/JMeterMigration.scala
Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/JMeterMigration.scala]: Fixed

**Review by eddumelendez — DISMISSED** [review]

**Comment by marc0der** [comment]
Apologies, this branch needs a rebase to remove a defunct migration @xshyamx

**Comment by xshyamx** [comment]
Bumped up changelog to 51 with latest repo changes

**Review by marc0der — APPROVED** [review]

**Comment by marc0der** [comment]
Thanks for contributing @xshyamx!

**Pull request merged**
